### PR TITLE
Encrypted guest root now resizes itself to fit guest

### PR DESCRIPTION
**Pull request by robertabbott**
The disk to which the encrypted guest is dd'd is now
2x + 1 the size of the guest root instead of always
defaulting to 25GB

Testing: encrypted ubuntu-14.04, centos6 and rhel6
images and verified the guest booted

### CODE REVIEW

File: brkt_cli/encrypt_gce_image.py [hunk 1]
```diff
@@ -397,8 +405,9 @@ def encrypt(gce_svc, enc_svc_cls, image_id, encryptor_image,
                                 encryptor,
                                 encrypted_image_disk])
 
-        log.info("Image %s successfully created!", encrypted_image_name)
-        return encrypted_image_name
+        if success:
+            log.info("Image %s successfully created!", encrypted_image_name)
```
Comment (bburtin) [inline brkt_cli/encrypt_gce_image.py]: Can't you just do this at the line that says `success = True`?  Then you wouldn't need the variable at all.

Comment (robertabbott) [inline brkt_cli/encrypt_gce_image.py]: that makes more sense.

Comment (robertabbott) [inline brkt_cli/encrypt_gce_image.py]: fixed

**Comment by bburtin** [comment]
Looks good.

**Pull request merged**